### ISSUE TITLE
Firefox font change when opening web terminal

### ISSUE THREAD

**Issue by Kira-Pilot** [coder/coder#1921]
When I open the web terminal on Firefox, I am presented with this bizarre font choice:
![Screen Shot 2022-05-31 at 11.03.35 AM.png](https://images.zenhubusercontent.com/62791b32ea910974a977e799/1b81a842-3f26-4f02-8aed-f59c495480c6)
The font flickers as I type, switching between this unexpected face and the expected face. If I resize the window, the font returns to normal.

**Comment by kylecarbs** [comment]
@Kira-Pilot I see this is on `master.cdr.dev`, are you sure this happens in v2?

**Comment by Kira-Pilot** [comment]
@kylecarbs you're right, you're right!!! I just realized I'm dogfooding on V1. My bad - is this worth a bug in shortcut or should I just ignore?

**Comment by kylecarbs** [comment]
I'd say a bug is good just to track!

**Comment by Kira-Pilot** [comment]
Closing in favor of 23278 in Shortcut.
